### PR TITLE
Project carousel

### PR DESCRIPTION
**Pull request by jaynopponep**
- built blueprint for project carousel
- fixed height of main component and removed unnecessary empty space
- redirected navbar items Projects and About to their corresponding components
- adjusted color scheme of main component
- added name/title of project, description, and image to each project carousel slide
- thumbnail/buttons work in which clicking causes a live slide-redirect on the carousel
stylistic improvisations for the future:
- limiting amount of horizontal scrolling
- changing orientation to reflect original design planned on Figma
- letting users see more about each project with a "see more" popup

### CODE REVIEW

**Comment by jaynopponep** [comment]
project_carousel
Here's the demo of the progress!

**Comment by LordFarquaadtheCreator** [comment]
looks great, however the images aren't scrollable - draggable is nice but we need slideshows to be scrollable. 
Also, it's not updated with main so it doesn't have Firefox compatibility. You can simply do `git merge origin/main` to fix this (make sure local main is up to date).

**Comment by jaynopponep** [comment]
i think we're good to go, fixed merge conflicts

**Comment by jaynopponep** [comment]
@LordFarquaadtheCreator do you want me to implement scrolling before merging this PR or after? id figured after since this branch is kind of behind with all changes to main

**Comment by LordFarquaadtheCreator** [comment]
> @LordFarquaadtheCreator do you want me to implement scrolling before merging this PR or after? id figured after since this branch is kind of behind with all changes to main

you can do it in another pr, there are some nitpicks here and there but the core functionality is solid.

File: src/app/projects/page.tsx
Comment (LordFarquaadtheCreator) [inline src/app/projects/page.tsx:1]: this should be a usestate - it might end up doing hundreds of fetches per second

**Review by LordFarquaadtheCreator — APPROVED** [review]
peep the comment about usestate - keep it in mind for the next pr

**Pull request merged**